### ISSUE TITLE
Bard buttons showing, when all are deselected

### ISSUE THREAD

**Issue by goellner** [statamic/cms#1486]
After deselecting all bard buttons, then the defaults are shown, since it is not saving anything in the fieldset config. 

Also a select/deselect all button would be nice in the fieldset builder.

**Comment by jackmcdade** [comment]
The problem with this one is that if we support unsetting all buttons, you will have to explicitly define all buttons if you're working in the YAML directly. Anyone relying on the default button set will suddenly find themselves with no buttons after updating.

**Comment by goellner** [comment]
Or it saves something like „none“

**Comment by jelleroorda** [comment]
I think Goellners idea would be a fine workaround to prevent a breaking change.

**Comment by jasonvarga** [comment]
Just confirming you're wanting a Bard field with no buttons?

**Comment by jelleroorda** [comment]
I'm not sure whether I'd need a bard editor without buttons anymore... I can't remember what I needed it for at the moment. I vaguely remember 'needing it' when I just started using Statamic, but now you put it like this I guess the usecase has been solved with a combination of other fields / read_only fields.

**Comment by jasonvarga** [comment]
Okay. Feel free to comment if you can explain the reason for needing no buttons.

**Comment by jelleroorda** [comment]
Not sure, since if you just need plain text you might as well just use a textarea field?

**Comment by goellner** [comment]
Let me give you one quick example I could think of: 

A bard field with the option to add plain text AND bard sets. Thats not possible with a textarea.

**Comment by jasonvarga** [comment]
It's not really plain text though. You'd have `<p>` tags in it since Bard saves html.

What about using replicator with a `text` set containing a textarea?

**Comment by goellner** [comment]
Thats correct, tbh I don't remember exactly why I needed that, but I created the issue, because it would be just a nice option to have.
